docs/src/pipe.rst

```diff
@@ -102,3 +102,12 @@ API
     and call ``uv_accept(pipe, handle)``.
 
 .. seealso:: The :c:type:`uv_stream_t` API functions also apply.
+
+.. c:function:: int uv_pipe_chmod(uv_pipe_t* handle, int flags)
+
+    Alters pipe permissions, allowing it to be accessed from process running by
+    different users. Makes the pipe writable or readable by all users. Mode can
+    be either ``UV_WRITABLE``, ``UV_READABLE`` or
```

Comment:
I'd word this sentence as:

Mode can be `UV_WRITABLE`, `UV_READABLE`, or `UV_WRITABLE | UV_READABLE`.